NitroxServer/ConsoleCommands/TimeCommand.cs

```diff
@@ -9,15 +9,16 @@ internal class TimeCommand : Command
     {
         private readonly TimeKeeper timeKeeper;
 
-        public TimeCommand(TimeKeeper timeKeeper) : base("time", Perms.ADMIN, "Changes the map time")
+        public TimeCommand(TimeKeeper timeKeeper) : base("time", Perms.MODERATOR, "Changes the map time")
         {
-            this.timeKeeper = timeKeeper;
             AddParameter(new TypeString("day/night", false));
+
+            this.timeKeeper = timeKeeper;
         }
 
         protected override void Execute(CallArgs args)
         {
-            string time = args.Get(0);
+            string time = args.Get<string>(0);
```

Comment:
Not sure why this change is done. If we want to require specifying the type we should remove the method `public string Get()` in CallCargs.cs that makes this possible.